config/routes.rb

```diff
@@ -12,10 +12,12 @@
   resources :users, only: [:show, :update]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
-
+  get '/videos' => 'users#videos', as: '/videos'
   # You can have the root of your site routed with "root"
   # root 'welcome#index'
   root 'home#index'
+  
+  #get 'playstv', to: 'home#playstv'
```

Comment:
Do we need this route or is it suppose to be commented out?

Reply:
It is working with out it, but I am not sure if we may need it in the futrue, so I thought I would leave commented out for now.

Reply:
👍